### DISCUSSION TITLE
post template for specific tags is a copy of index.hbs, but does not masonry

### DISCUSSION THREAD

**Discussion by michaelfluegge** [GetPublii/Publii#1879] [Help] [answered]
For my site I built a post template based on index.hbs to show specific posts, filtered by a tag. Everything works as expected except the fact, that the sub-post which shows the posts with the specific tag does not order the featured images as masonry as  the homepage does :-(
I did not find any explanation neither on the forum nor the documentation.

The only difference to the index.hbs of the homepage to the sub-post.hbs is the code suggested on https://getpublii.com/dev/how-to-display-posts-connected-with-a-specific-tag-name/

One can see the difference on these screenshots down below. 

An example is 
homepage: https://fluegge.net/unterwegs/
sub-post: https://fluegge.net/unterwegs/mallorca/

homepage:
![Bild 15 12 24 um 16 51](https://github.com/user-attachments/assets/9b876985-e62c-4cd0-9990-db4d3488c2bb)

sub-post:
![Bild 15 12 24 um 16 53](https://github.com/user-attachments/assets/ad1cddc4-01ea-4d85-adbd-6ffb7ddf9b09)

**Comment by bobmitro** [comment, marked as answer]
Check the head.hbs and footer.hbs files to find the masonry script. There must be a condition that limits the script's functionality to the index only. You need to extend this also to include it on the page and post or to a post-template.

**Reply by michaelfluegge** [reply]
Woah, that was quick.

**Reply by michaelfluegge** [reply]
Thanks for your help :-), @bobmitro

**Comment by michaelfluegge** [comment]
Just in case anyone else has the same problem, I tried to extend my 'documentation': https://fluegge.net/ohnein/how-to-create-a-post-template-which-fetches-posts-by-tag/
